exporter/splunkhecexporter/client_test.go

```diff
@@ -1513,6 +1513,44 @@ func TestAllowedLogDataTypes(t *testing.T) {
 	}
 }
 
+func Test_heartbeat_success(t *testing.T) {
```

Comment:
This test is pretty similar to `Test_Heartbeat_success` in heartbeat_test. If we merge `newHeartbeater` and `initHeartbeat`, I think we can remove it along with `getPushLogFn`